src/Adapter/Phpunit/Process/PhpunitExecutableFinder.php

```diff
@@ -104,10 +104,14 @@ protected function makeExecutable($path)
         if (!defined('PHP_WINDOWS_VERSION_BUILD')) {
             return sprintf('%s %s %s', 'exec', $phpFinder->find(), $path);
         } else {
-            if (false !== strpos($path, '.bat')) {
+            if (false !== stripos($path, '.bat')) {
                 return $path;
             }
-            return sprintf('%s %s', $phpFinder->find(), $path);
+            /*
+             * Was: return sprintf('%s %s', $phpFinder->find(), $path);
```

Comment:
Mainly just a memory aid. I assume there was some Windows variant where this actually worked, so that does crop up as an issue, might save some time debugging. For now through, the `php` command that was there really shouldn't be needed since the phpunit command is an executable script on Powershell.